package.json

```diff
@@ -2,11 +2,6 @@
   "private": true,
   "scripts": {
     "build": "bower install && pulp build",
-    "documentation": "cd example && pulp build -O -m Main -t html/test.js",
```

Comment:
I think we can delete this file entirely since this library doesn’t use this file in CI